pkg/abi/nvgpu/ctrl.go

```diff
@@ -90,12 +90,19 @@ type NV0080_CTRL_FIFO_GET_CHANNELLIST_PARAMS struct {
 
 // From src/common/sdk/nvidia/inc/ctrl/ctrl0080/ctrl0080gpu.h:
 const (
+	NV0080_CTRL_CMD_GPU_GET_CLASSLIST              = 0x800201
 	NV0080_CTRL_CMD_GPU_GET_NUM_SUBDEVICES         = 0x800280
 	NV0080_CTRL_CMD_GPU_QUERY_SW_STATE_PERSISTENCE = 0x800288
 	NV0080_CTRL_CMD_GPU_GET_VIRTUALIZATION_MODE    = 0x800289
 	NV0080_CTRL_CMD_GPU_GET_CLASSLIST_V2           = 0x800292
 )
 
+// +marshal
+type NV0080_CTRL_CMD_GPU_GET_CLASSLIST_PARAMS struct {
+	NumClasses	uint32
+	ClassList	P64
```

Comment:
there is a uint32 padding after `NumClasses`. Note that this field is aligned: https://github.com/NVIDIA/open-gpu-kernel-modules/blob/be3cd9abcb1103115ae6c3c92d8fc4ff5c912f77/src/common/sdk/nvidia/inc/ctrl/ctrl0080/ctrl0080gpu.h#L76C12-L76C12

Reply:
Oh thanks I totally misread that, thought it was aligned 32. Will fix.